src/3d/qgs3dmapexportsettings.h.bom

```diff
@@ -0,0 +1,45 @@
+﻿/***************************************************************************
```

Comment:
looks like this file has been added by mistake?

Reply:
Yes I deleted the file